frontend/src/components/Header.tsx

```diff
@@ -1,79 +1,113 @@
-import { faMoon, faSun } from '@fortawesome/free-regular-svg-icons'
```

Comment:
Why have we changed this? 🤔 We kinda wanted to have these sun and moon icons for dark and light theme. Can we go back to a previous view that used these icons? The `ModeToggle` component is not something we use right now, but we'd like to keep to update later on.

Reply:
I have re-added the sun and moon icon in the header